lack/src/main/java/io/datanerds/lack/Lack.java

```diff
@@ -49,4 +57,16 @@ public void stop() {
         session.close();
         client.close();
     }
+
+    private void checkLockOwner(String resource,  ResultSet result) throws LackException {
```

Comment:
This method should be called something like "renewIfOwned" to emphasize that it also performs an action. I'd expect a check to return a boolean or only throw an exception.